framework/src/main/java/org/checkerframework/common/value/statically-executable.astub

```diff
@@ -334,6 +334,57 @@ class String{
     @StaticallyExecutable static String valueOf(Object obj) ;
 }
 
+package java.math;
+
+class BigInteger {
+    @StaticallyExecutable BigInteger(@IntRange(from = -1, to = 1) int signum, byte[] magnitude);
+    @StaticallyExecutable BigInteger(byte[] val);
+    @StaticallyExecutable BigInteger(int bitLength, int certainty, Random rnd);
+    @StaticallyExecutable BigInteger(int numBits, Random rnd);
+    @StaticallyExecutable BigInteger(String val);
+    @StaticallyExecutable BigInteger(String val, @IntRange(from = 2, to = 36) int radix);
+
+    @StaticallyExecutable BigInteger abs();
+    @StaticallyExecutable BigInteger add(BigInteger val);
+    @StaticallyExecutable BigInteger and(BigInteger val);
+    @StaticallyExecutable BigInteger andNot(BigInteger val);
```

Comment:
Similar to the constructors, these methods cannot be `@StaticallyExecutable`: they allocate new objects.

Reply:
In fact, no methods that return a non-primitive type in this class are statically executable. You should remove `@StaticallyExecutable` from them all.